app/src/components/testCaseList/index.js

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2024 EPAM Systems
```

Comment:
```suggestion
 * Copyright 2025 EPAM Systems
```

please, adjust in the other places as well

Reply:
renamed